### PR TITLE
Daml Finance / version update

### PR DESCRIPTION
**Pull request by johan-da**
This PR:
1. Adds a comment about the `daml-version`.
2. Makes some minor style changes.

### CODE REVIEW

File: docs/3.1/versions.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "daml": "2.8.0-snapshot.20231118.12382.0.v86cb8054",
   "canton": "2.9.0-snapshot.20240402.11920.0.v2507c677",
-  "daml_finance": "1.4.1",
```
Comment (matteolimberto-da) [inline docs/3.1/versions.json:4]: Do we already have an assembly for this?

File: docs/2.10.0/README.md [hunk 2]
```diff
@@ -10,9 +10,9 @@ This folder manages the full build of technical documentation from the following
 
 The process for updating the docs TOCs has some potential pitfalls. Follow the instructions in the main README *carefully* and in order.
 
-- [Setting up this repo locally](https://github.com/digital-asset/docs.daml.com#setting-up-this-repo-locally)
-- [Build and view the docs locally](#live-preview)
-- [How to commit changes to the docs](#how-to-commit-changes-to-the-docs)
+* [Setting up this repo locally](https://github.com/digital-asset/docs.daml.com#setting-up-this-repo-locally)
```
Comment (carrielaben-da) [inline docs/2.10.0/README.md:13]:
```suggestion
* [Set up this repo locally](https://github.com/digital-asset/docs.daml.com#setting-up-this-repo-locally)
```

File: docs/2.10.0/README.md [hunk 3]
```diff
@@ -10,9 +10,9 @@ This folder manages the full build of technical documentation from the following
...
+* [Build and view the docs locally](#live-preview)
+* [How to commit changes to the docs](#how-to-commit-changes-to-the-docs)
```
Comment (carrielaben-da) [inline docs/2.10.0/README.md:15]:
```suggestion
* [Commit changes to the docs](#how-to-commit-changes-to-the-docs)
```

**Review by matteolimberto-da — APPROVED** [review]

File: docs/2.9.1/README.md [hunk 4]
```diff
@@ -10,9 +10,9 @@ This folder manages the full build of technical documentation from the following
 
 The process for updating the docs TOCs has some potential pitfalls. Follow the instructions in the main README *carefully* and in order.
 
-- [Setting up this repo locally](https://github.com/digital-asset/docs.daml.com#setting-up-this-repo-locally)
-- [Build and view the docs locally](#live-preview)
-- [How to commit changes to the docs](#how-to-commit-changes-to-the-docs)
+* [Setting up this repo locally](https://github.com/digital-asset/docs.daml.com#setting-up-this-repo-locally)
+* [Build and view the docs locally](#live-preview)
+* [How to commit changes to the docs](#how-to-commit-changes-to-the-docs)
```
Comment (carrielaben-da) [inline docs/2.9.1/README.md:15]:
```suggestion
* [Set up this repo locally](https://github.com/digital-asset/docs.daml.com#setting-up-this-repo-locally)
* [Build and view the docs locally](#live-preview)
* [Commit changes to the docs](#how-to-commit-changes-to-the-docs)
```

**Review by carrielaben-da — APPROVED** [review]

**Pull request merged**